packages/tupaia-web/src/layout/ProjectCardList/ProjectCard.tsx

```diff
@@ -64,6 +65,7 @@ const Text = styled(Typography)`
   font-size: 0.875rem;
   line-height: 1.2;
   margin-bottom: 0.625rem;
+  color: ${({ theme }) => theme.palette.text.primary};
```

Comment:
Just double checking that these are the right changes? Or are they out of date now?

Reply:
Oh, I think I accidentally added this in when it was meant to be for the other project maybe... I have now removed it